tools/nuget/template.nuspec

```diff
@@ -21,7 +21,7 @@
         <projectUrl>https://github.com/onnx/onnxruntime</projectUrl>
 
         <!-- The icon is used in Visual Studio's package manager UI -->
-        <iconUrl>http://github.com/contoso/UsefulStuff/nuget_icon.png</iconUrl>
+        <icon>http://github.com/contoso/UsefulStuff/nuget_icon.png</icon>
```

Comment:
Does it support a url? We may need to put the png file in the package.

Reply:
oh yes you're right. i'm not familiar with the nuspec templates...will i be able to directly reference the icon under docs/images/ONNX_Runtime_icon.png?

Reply:
No.

Example:
```xml
<package>
  <metadata>
    ...
    <icon>images\icon.png</icon>
    ...
  </metadata>
  <files>
    ...
    <file src="..\icon.png" target="images\" />
    ...
  </files>
</package>
```

We need to put it the files section first. And that section is generated from 
 tools\nuget\generate_nuspec_for_native_nuget.py function `generate_files`